pylint/pyreverse/diagrams.py

```diff
@@ -3,8 +3,14 @@
 # Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
 
 """Diagram objects."""
+from __future__ import annotations
+
+from collections.abc import Iterable
+from typing import Any
+
 import astroid
 from astroid import nodes
+from astroid.nodes import ClassDef, FunctionDef, ImportFrom, Module, NodeNG
```

Comment:
We already import `nodes` above. Did you explicitly want to use the shorthand versions?

Reply:
Well, more a "have to" instead of "want to": `ClassDiagram` defines a method named `nodes`, and after this method declaration you can't use the imported `nodes` from `astroid` any more in the scope of this class definition.
But I can also remove the `nodes` method - from what I've seen so far it is never used anyways.

Reply:
Removed the `nodes()` method in the latest commit and use `nodes` instead of shorthand versions.